melodic/distribution.yaml

```diff
@@ -6198,6 +6198,21 @@ repositories:
       url: https://github.com/nobleo/mobile_robot_simulator.git
       version: master
     status: maintained
+  mocap_optitrack:
+    doc:
+      type: git
+      url: https://github.com/clearpathrobotics/mocap_optitrack.git
```

Comment:
I missed this before, but this is pointing to the old name of the repository.  Can you update this to `https://github.com/ros-drivers/mocap_optitrack.git`?  Also, I'd appreciate it if you did the same for Kinetic.

Reply:
Sorry, I missed that.  Updated for both.